### PR TITLE
Added bower.json

### PR DESCRIPTION
**Pull request by febLey**
https://github.com/typekit/webfontloader/issues/302

Credits to https://github.com/alepee

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "webfontloader",
+  "main": "webfontloader.js",
+  "version": "1.6.21",
```
Comment (bramstein) [inline bower.json]: Is there any chance we could do without the version number? If not, we should change the release script so it auto updates the version on release.

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "homepage": "https://github.com/typekit/webfontloader",
+  "authors": [
```
Comment (bramstein) [inline bower.json]: Do we really need to duplicate this stuff? It's already in the package.json file.

**Comment by gjungb** [comment]
@bramstein actually the ```version``` field has been deprecated, so there is no need for using it (https://github.com/bower/spec/blob/master/json.md#version)

**Comment by abdonrd** [comment]
Any updates on this?

**Pull request merged**